code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -0,0 +1,43 @@
+/datum/customer_data
+	///Name of the robot by default. Can be overriden at runtime
+	var/list/name = "Generic Robot"
+	///The types of food this robot likes in a assoc list of venue type | weighted list. does NOT include subtypes.
+	var/list/orderable_objects = list()
+	///The amount a robot pays for each food he likes in an assoc list type | payment
+	var/list/order_prices = list()
+	///Datum AI used for the robot. Should almost never be overwritten unless theyre subtypes of ai_controller/robot_customer
+	var/datum/ai_controller/ai_controller_used = /datum/ai_controller/robot_customer
+	///Patience of the AI, how long they will wait for their meal.
+	var/total_patience = 600 SECONDS
+	///Lines the robot says when it finds a seat
+	var/list/found_seat_lines = list("I found a seat")
+	///Lines the robot says when it can't find a seat
+	var/list/cant_find_seat_lines = list("I did not find a seat")
+	///Lines the robot says when leaving without food
+	var/list/leave_mad_lines = list("Leaving without food")
+	///Lines the robot says when leaving with food
+	var/list/leave_happy_lines = list("Leaving with food")
+	///Lines the robot says when leaving waiting for food
+	var/list/wait_for_food_lines = list("I'm still waiting for food")
+	///Clothing sets to pick from when dressing the robot.
+	var/list/clothing_sets = list("amerifat_clothes")
+
+/datum/customer_data/american
+	name = "American Robot"
```

Comment:
Is America cannon?

Reply:
america is artillery

Reply:
![image](https://user-images.githubusercontent.com/29008542/108296103-ea832c00-7166-11eb-967e-e74af4841ced.png)

Reply:
but on an unironic note, we have a bunch of american references already like that one vendor that sells weapons and has a huge american flag on it

Reply:
Dear god

Reply:
Space-America is canon

Reply:
I thought it was corporations that owned space? I'm not into that whole lorebus thing. America obviously existed at one point, though.